src/NServiceBus.Core/Hosting/HostInfoSettings.cs

```diff
@@ -67,6 +68,16 @@ public HostInfoSettings UsingCustomDisplayName(string displayName)
             return this;
         }
 
+        /// <summary>
+        /// Allows overriding the host name of the endpoint.
+        /// </summary>
+        public HostInfoSettings OverrideHostName(string hostName)
```

Comment:
Based on the other API's shouldn't this be `UsingHostName`?

The Using methods seem to be related to the ID, this isn't affecting the ID so I think its fine.

Reply:
Renamed